registry/storage/driver/azure/azure.go

```diff
@@ -73,27 +88,71 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		realm = azure.DefaultBaseURL
 	}
 
-	return New(fmt.Sprint(accountName), fmt.Sprint(accountKey), fmt.Sprint(container), fmt.Sprint(realm))
+	fixleadingslashBool := false
+	fixleadingslash, ok := parameters[paramFixLeadingSlash]
+	if !ok {
+		fixleadingslash = false
+	}
+	switch fixleadingslash := fixleadingslash.(type) {
```

Comment:
just suggestion, we can add a unility func for driver, maybe others have the same requirement.

Reply:
I would rename the parameter to match the gitlab changes.

My logic for default was off (to not break existing installs) but on if a prefix was defined (since that is not available for existing installs anyway).